README.txt

```diff
@@ -21,9 +21,14 @@ This module requires the following modules:
 INSTALLATION
 ------------
 
+This module uses Composer to install dependencies. You can either execute
```

Comment:
Can we save the README update for https://github.com/BurdaMagazinOrg/module-fb_instant_articles/pull/19? I note more info about Composer options, and try to keep it simple there. Although I could add this link in part of the note that's already there.